src/awx_plugins/interfaces/_temporary_private_credential_api.py

```diff
@@ -0,0 +1,58 @@
+# pylint: disable=fixme
+"""Shared stubs from ``awx`` credential.
+
+The hope is that it will be refactored into something more standardized.
+"""
+
+from typing import Union
+
+
+GenericOptionalPrimitiveType = Union[bool, str, int, float, None]
```

Comment:
Can't this use newer syntax?
```suggestion
GenericOptionalPrimitiveType = bool | str | int | float | None
```

Reply:
```
  10:32    WPS465 Found likely bitwise and boolean operation mixup
           -> https://pyflak.es/WPS465
  GenericOptionalPrimitiveType = bool | int | str | float | None
```

Reply:
FTR my solution was noqa